sdk/tables/Azure.Data.Tables/README.md

```diff
@@ -213,7 +230,7 @@ tableClient.DeleteEntity(partitionKey, rowKey);
 
 ## Troubleshooting
 
-When you interact with the Azure table library using the .NET SDK, errors returned by the service correspond to the same HTTP 
+When you interact with the Azure table library using the .NET SDK, errors returned by the service correspond to the same HTTP
```

Comment:
```suggestion
When using the Azure tables library, errors returned by the service are reported using the same HTTP
```

Reply:
This read awkwardly to me.   I'm not sure that I'm in love with my suggestion either, but the "Azure tables library" and ".NET SDK" distinction was where I struggled a bit and had to read a few times to understand.